src/main/app/case/definition.ts

```diff
@@ -161,8 +161,8 @@ export interface CaseData {
   applicant1KnowsApplicant2Address: YesOrNo;
   applicant2HomeAddress: AddressGlobalUK;
   legalProceedings: YesOrNo;
-  legalProceedingsDetails: string;
-  legalProceedingsRelated: LegalProceedingsRelated[];
+  legalProceedingsByCase: ListValue<LegalProceeding>[];
+  legalProceedingsOther: string;
```

Comment:
Won't the implementations of these fields need their name changing too?

Reply:
yep

Reply:
its become more complex hence i closed this pr. Whats it requires is more than what we can give so i closed it as Tom (who made the changes) said he'll look into it.